types/ckeditor__ckeditor5-autoformat/index.d.ts

```diff
@@ -1,6 +1,12 @@
-// Type definitions for @ckeditor/ckeditor5-autoformat 27.0
+// Type definitions for @ckeditor/ckeditor5-autoformat 27.1
 // Project: https://ckeditor.com/docs/ckeditor5/latest/api/autoformat.html
 // Definitions by: Federico Panico <https://github.com/fedemp>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // Minimum TypeScript Version: 4.2
-export { default as Autoformat } from "./src/autoformat";
+import Autoformat from './src/autoformat';
+
+declare const _default: {
+    Autoformat: typeof Autoformat;
+};
+
+export default _default;
```

Comment:
```suggestion

export { Autoformat };
```
https://github.com/ckeditor/ckeditor5/blob/master/packages/ckeditor5-autoformat/src/index.js#L10